.gitlint

```diff
@@ -0,0 +1,12 @@
+[general]
+ignore=T5
```

Comment:
Could you add `contrib=contrib-title-conventional-commits,CC1` here and remove it from the action?
```suggestion
contrib=contrib-title-conventional-commits,CC1
ignore=T5
```